documentation/modules/proc-create-address-console.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='create-address-console-{context}']
+= Creating addresses using the {ProductName} Console
+
+You can create new addresses using the {ProductName} Console. The type of addresses that you can create are determined by the type of address space. For more information see the xref:ref-supported-features-table[].
```

Comment:
ref-supported-features-table does not resolve

Reply:
Fixed